src/signals/incident-management/containers/IncidentDetail/components/StatusForm/styled.tsx

```diff
@@ -2,85 +2,114 @@
 // Copyright (C) 2020 - 2021 Gemeente Amsterdam
 import styled from 'styled-components'
 import {
+  Alert,
   Heading,
-  themeSpacing,
+  Label,
   Row,
-  Column,
   themeColor,
-  Label,
+  themeSpacing,
 } from '@amsterdam/asc-ui'
 
 import Button from 'components/Button'
 
+export const AddNoteWrapper = styled.div`
+  label {
+    display: none;
+  }
+  section div textarea {
```

Comment:
This is a very unspecific selector and is very likely to break when something in the layout of the `AddNote` component is changed. Can this selector be made more specific, for instance by targeting an idea or class name?

Reply:
done

Reply:
Shouldn't this rule be removed as well, then?